src/ApiPort/ConsoleProgressReporter.cs

```diff
@@ -83,35 +131,53 @@ public ConsoleProgressTask(string task, int? totalCount)
                 await Task.Delay(1);
 
 #if FEATURE_RICH_CONSOLE
```

Comment:
I don't think we need this define anymore... it comes in .NET Core too. I'll remove it.